lib/platform/git/storage.ts

```diff
@@ -183,7 +184,13 @@ export class Storage {
       const cloneStart = process.hrtime();
       try {
         // clone only the default branch
-        await this._git.clone(config.url, '.', ['--depth=2']);
+        let opts = ['--depth=2'];
+        if (config.extraCloneOpts) {
+          opts = opts.concat(
+            Object.entries(config.extraCloneOpts).map(e => `${e[0]}=${e[1]}`)
```

Comment:
Missing `--` from extra options?

```suggestion
            Object.entries(config.extraCloneOpts).map(e => `--${e[0]}=${e[1]}`)
```

Reply:
The "--" is currently located on this line [here](https://github.com/kroonprins/renovate/blob/ac2cb86d409270e3a16493cf1e9e11fb3f401d6b/lib/platform/azure/azure-helper.ts#L34). I could remove it there and move it here if you prefer